metasync.js

```diff
@@ -1,5 +1,8 @@
 'use strict';
 
+const common = require('metarhia-common');
```

Comment:
This package does not depend on `metarhia-common` package directly. This may lead to this code breaking on the next dependency update.
https://github.com/metarhia/metasync/blob/8849e73793efdb79e009a29ba193a99616570c56/package.json#L48-L50